testnet/stacks-node/conf/xenon-miner-conf.toml

```diff
@@ -9,9 +9,12 @@ miner = true
 [burnchain]
 chain = "bitcoin"
 mode = "xenon"
-peer_host = "xenon.blockstack.org"
+peer_host = "127.0.0.1"
```

Comment:
Are we setting up some public testnet nodes?

Reply:
I'm not sure, but in my mind the main point of xenon is to make it so community members run their own Bitcoin node.